akka-sample-sharding-scala/src/main/scala/sample/sharding/TemperatureService.scala

```diff
@@ -0,0 +1,65 @@
+package sample.sharding
+
+import scala.concurrent.duration._
+import scala.util.Random
+
+import akka.actor.typed.{ ActorRef, Behavior }
+import akka.actor.typed.scaladsl.Behaviors
+import akka.cluster.sharding.typed.{ ClusterShardingSettings, ShardingEnvelope }
+import akka.cluster.sharding.typed.scaladsl.{ ClusterSharding, Entity, EntityTypeKey }
+import akka.cluster.typed.{ Cluster, Join }
+
+object TemperatureService {
+
+  // Update a random device
+  case object UpdateDevice extends Message
```

Comment:
isn't `Message` just some marker interface for serialization?
define more specific `Command` for the `TemperatureService`